exasol_advanced_analytics_framework/udf_communication/peer_communicator/connection_establisher.py

```diff
@@ -48,51 +37,24 @@ def __init__(self,
         self._send_initial_messages()
 
     def _send_initial_messages(self):
-        self._register_peer_sender.try_send(force=True)
-        self._acknowledge_register_peer_sender.try_send(force=True)
         self._synchronize_connection_sender.try_send(force=True)
 
     def received_synchronize_connection(self):
         self._logger.debug("received_synchronize_connection")
-        self._peer_is_ready_sender.received_synchronize_connection()
-        self._peer_is_ready_sender.reset_timer()
-        self._abort_timeout_sender.received_synchronize_connection()
-        self._sender.send(Message(__root__=AcknowledgeConnection(source=self._my_connection_info)))
+        self._sender.send(Message(__root__=messages.AcknowledgeConnection(source=self._my_connection_info)))
+        self._connection_is_ready_sender.received_synchronize_connection()
```

Comment:
Just an observation:
When I see variable name `_sender`, and also `_connection_is_ready_sender`, it's not obvious what `_sender` means.

Reply:
Also, the order of invocation of `send()` was changed in this change. Is this ok?

Reply:
The order doesn't really matter, because the _sender sends messages to the other peer and the connection_is_ready_sender informs the foreground thread.

However, you are right, the name _sender isn't really helpful. Maybe we should call it peer_sender or something is this direction, because it is basically the way to send messages to another peer.